docs/json/radarr/cf/hdr10.json

```diff
@@ -19,7 +19,7 @@
       "negate": true,
       "required": true,
       "fields": {
-        "value": "^(?=.*\\b(HDR(10)?)\\b)(?=.*\\b(DV|DoVi)\\b)"
+        "value": "^(?=.*\\b(HDR(10)?(?!\\+))\\b)(?=.*\\b(DV|DoVi)\\b)"
```

Comment:
why is this needed?

"name": "Not HDR10+" prevents this

Reply:
we changed the CF a while ago to make it better matching, and to make it easier to update we prefer to keep the conditions consistent.
but we don't want to go fully one liners